app/controllers/person.js

```
@@ -48,6 +49,34 @@ class Person {
+        if (!characterId) {
+            return res.status(400).json({ error: 'Missing required fields' });
+        }
+
+        try {
+            const character = await Character.findById(characterId);
+
+            if (!character) {
+                return res.status(404).json({ error: 'Character not found' });
+            }
+
+            const { link } = character?.image;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)